### ISSUE TITLE
Overlay invisible (Multi monitor, negative coordinates)

### ISSUE THREAD

**Issue by LinusSte** [Manuel-777/MTG-Arena-Tool#92]
I have the issue that I cannot see the overlay, no matter what.
I can tell it is there only from the windows task bar, where a second icon of your application appears. When I hover the cursor above it, I also can see that there is something in there, as I can see the small preview as usually. However, no matter what I do it does not appear as an actual window.
Clicking on it does nothing, I even thought it would be offscreen, but it still does not appear when I use the rearrange windows feature from windows. The problem persisted even after uninstalling and reinstalling, boh on .15 and .17.

**Comment by Manuel-777** [comment]
Hi!
Please follow the troubleshooting guide;
https://github.com/Manuel-777/MTG-Arena-Tool/blob/master/TROUBLESHOOTING.md

I would like a sample of your config file precisely, and a screenshot would be helpful too!
Thanks!

**Comment by IanWhalen** [comment]
I'm seeing this too.  I've tried following the troubleshooting guide but to no avail.  Which file is the config file?  default.json?  something else?

**Comment by LinusSte** [comment]
I feel stupid that I have not thought about your troubleshooting guide. I deleted the MTGA logs, but did not think about the config file. Which, in this case, was the issue.
Attached the config file that resulted in the invisible window. A screenshot doesn't make sense though, since you literally see nothing from the overlay other than the preview in the task bar, which looked completely normal and also was updating. Main application always worked fine.
[BOOZMBF56JCIVFZNKZUZ6BDSWU.zip](https://github.com/Manuel-777/MTG-Arena-Tool/files/2564306/BOOZMBF56JCIVFZNKZUZ6BDSWU.zip)

**Comment by Manuel-777** [comment]
Ah okay, issue was simply the overlay window went somewhere you cant see, if you would like to maintain your previous history and stuff (and I assume you want that), do not delete the old .json config. Open in a text editor and you will see something like this;

`	"overlayBounds": {
		"height": 1044,
		"width": 612,
		"x": 2052,
		"y": -1896
	},`

Simply change the "Y" value to zero, save and try again. I have no idea how this could happen trough, unless you use many monitors or have a weird display configuration :/

**Comment by LinusSte** [comment]
I had a look at the json file and noticed there basically only is the "overlayBounds" section that could be related to the issue. When I replaced that section with what your app generates when it does not find a config, and that fixed the issue.

**Comment by LinusSte** [comment]
Ninja'd.
Yeah, I use two displays stacked vertically and moved the overlay to the upper screen, so it is not overlaying MTGA which is running on the lower screen. 
However, I did not change my display configuration, so it should still be visible there. Also I do think that there is something else, maybe in the way your application renders the overlay, as the overlay-window is not "simply" out of bounds.
Like I wrote in the first comment, it even did not appear when I used that rearrange windows function (or whatever it is called in english... in Windows 10, right-click on the task bar and you'll see what I mean in the context menu), which usually forces ALL windows to move to specific locations onscreen.
If it was just out of bounds, that should have made it visible, but it didn't.

**Comment by Manuel-777** [comment]
Overlay starts at coordinates 0,0 as default, maybe it got confused about where 0,0 was? I'm unsure as its something the app should do without intervetion.

**Comment by LinusSte** [comment]
0,0 in my case is the upper left corner of the lower screen. As I didn't have the issue from the beginning, I used the tool and intentionally moved the overlay to the right side of my upper screen, hence the negative Y coordinates.
When I then restarted the tool at a later point, I noticed the issue described here above. So I'd guess it might have to an issue with negative coordinates during the start-up?
